src/components/forms/ComboBox/ComboBox.test.tsx

```diff
@@ -0,0 +1,1111 @@
+        />
+      )
+
+      userEvent.type(getByTestId('combo-box-input'), 'a')
+      userEvent.tab()
+      fireEvent.keyDown(getByTestId('combo-box-option-apple'), {
+        key: 'ArrowDown',
+      })
+
+      expect(getByTestId('combo-box-option-apricot')).toHaveFocus()
+    })
+
+    it('focuses the previous option when up arrow is pressed', () => {
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          options={fruitOptions}
+          onChange={jest.fn()}
+        />
+      )
+
+      userEvent.type(getByTestId('combo-box-input'), 'a')
+      userEvent.tab()
+      fireEvent.keyDown(getByTestId('combo-box-option-apple'), {
+        key: 'ArrowDown',
+      })
+      fireEvent.keyDown(getByTestId('combo-box-option-apricot'), {
+        key: 'ArrowDown',
+      })
+      fireEvent.keyDown(getByTestId('combo-box-option-avocado'), {
+        key: 'ArrowUp',
+      })
+
+      expect(getByTestId('combo-box-option-apricot')).toHaveFocus()
+    })
+
+    it('opens the menu when down arrow is pressed in the input', () => {
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          options={fruitOptions}
+          onChange={jest.fn()}
+        />
+      )
+
+      fireEvent.focus(getByTestId('combo-box-input'))
+      fireEvent.keyDown(getByTestId('combo-box-input'), {
+        key: 'ArrowDown',
+      })
+
+      expect(getByTestId('combo-box-option-list')).toBeVisible()
+      expect(getByTestId('combo-box-option-apple')).toHaveFocus()
+    })
+
+    it('closes the menu when the first option is focused and arrow up is pressed', () => {
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          options={fruitOptions}
+          onChange={jest.fn()}
+        />
+      )
+
+      fireEvent.focus(getByTestId('combo-box-input'))
+      fireEvent.keyDown(getByTestId('combo-box-input'), {
+        key: 'ArrowDown',
+      })
+      fireEvent.keyDown(getByTestId('combo-box-option-apple'), {
+        key: 'ArrowUp',
+      })
+
+      expect(getByTestId('combo-box-option-list')).not.toBeVisible()
+      expect(getByTestId('combo-box-input')).toHaveFocus()
+    })
+
+    xit('does not change focus when last option is focused and down arrow is pressed', () => {
+      // TODO: 🐛 right now tabbing starts the list over at the beginning
+      const { getByTestId } = render(
+        <ComboBox
+          id="favorite-fruit"
+          name="favorite-fruit"
+          options={fruitOptions}
+          onChange={jest.fn()}
+        />
+      )
+
+      fireEvent.click(getByTestId('combo-box-input'))
+      fireEvent.focus(getByTestId('combo-box-option-yuzu'))
+      fireEvent.keyDown(getByTestId('combo-box-option-yuzu'), {
+        key: 'ArrowDown',
+      })
+
+      expect(getByTestId('combo-box-option-yuzu')).toHaveFocus()
```

Comment:
fwiw, in Chrome when I press the down arrow after the last option it does not change focus

Reply:
Yes. This documented in [a failing test](https://github.com/trussworks/react-uswds/blob/f6a08d3f59155e54c790f050c836a3826ff0c882/src/components/forms/ComboBox/ComboBox.test.tsx#L706-L724).